bin/downgrade

```diff
@@ -492,6 +523,9 @@ DOWNGRADE_MAXDEPTH=1
 DOWNGRADE_CONF="/etc/xdg/downgrade/downgrade.conf"
 DOWNGRADE_VERSION="11.3.0"
 DOWNGRADE_IGNORE="prompt"
+DOWNGRADE_TO_LATEST=0
+DOWNGRADE_TO_OLDEST=0
```

Comment:
I wonder if we should do `DOWNGRADE_SELECT=latest|oldest`, since `DOWNGRADE_LATEST=1 DOWNGRADE_OLDEST=1` is an invalid state. We could still use `--latest` and `--oldest` CLI options, but at least in terms of ENV have something a bit more robust. WDYT?

Reply:
They are not really env, they are just internal variables. I'll add some logic to make --latest and --oldest mutually exclusive -- this I think will be easier when using separate DOWNGRADE_TO_LATEST/DOWNGRADE_TO_OLDEST variables so I'd stick with them

Reply:
[7179aac](https://github.com/archlinux-downgrade/downgrade/pull/217/commits/7179aace86e6c841992052c6e450cfa6c8edbd82) makes it consistent with how DOWNGRADE_FROM_ALA and DOWNGRADE_FROM_CACHE work as "mutually exclusive".